src/Core/src/Handlers/ImageButton/ImageButtonHandler.Android.cs

```diff
@@ -33,9 +37,19 @@ protected override void ConnectHandler(AppCompatImageButton nativeView)
 		{
 			nativeView.Click += OnClick;
 			nativeView.Touch += OnTouch;
+
 			base.ConnectHandler(nativeView);
 		}
 
+		[MissingMapper]
+		public static void MapStrokeColor(IImageButtonHandler handler, IButtonStroke buttonStroke) { }
```

Comment:
Android implementation is missing, why? Because we have more than one good option.
- The first option would be to create a BorderButtonDrawable where to manage the border and apply the Drawable to **AppCompatImageButton**.
- The second option is to change the Handler in Android to use **MaterialButton** and thus be able to use the direct available properties (StrokeWidth etc) in the control.

What option do you think we should apply? Why?

Reply:
@jsuarezruiz I'd vote let's be consistent with Button and use the material button's properties like we do here: https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/Android/ButtonExtensions.cs#L9-L25